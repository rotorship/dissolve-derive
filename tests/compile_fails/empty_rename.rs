use dissolve_derive::Dissolve;

#[derive(Dissolve)]
struct EmptyRename {
	#[dissolved(rename = "")]
	field: String,
}

fn main() {}
