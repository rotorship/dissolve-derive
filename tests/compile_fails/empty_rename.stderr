error: unexpected end of input, expected identifier
 --> tests/compile_fails/empty_rename.rs:3:10
  |
3 | #[derive(Dissolve)]
  |          ^^^^^^^^
  |
  = note: this error originates in the derive macro `Dissolve` (in Nightly builds, run with -Z macro-backtrace for more info)
